### DISCUSSION TITLE
Admin links

### DISCUSSION THREAD

**Discussion by jkosick** [mealie-recipes/mealie#4212] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Links to admin settings screens are missing such as group/data/foods. I can find no link to take me there. 

### Please provide a concise description of the feature that would resolve your issue.

Provide menu links to all admin settings 

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Click on your profile and scroll down. You should see the Manage Data option there, if you're an admin or have the manage permission

**Reply by jkosick** [reply]
Thank you. My comment is to try and make it more intuitive. The settings menu link isn't clear when you are on the settings page on how to access that area. It seems strange to have to use the profile page to access an admin settings are and It would seem that having menu links in the settings menu would make it easier. Unless that is intentional?

**Reply by michael-genson** [reply]
I definitely think that makes sense, just wanted to point out that links to those pages do exist

**Reply by hay-kot** [reply]
> It seems strange to have to use the profile page to access an admin settings

These aren't admin settings, they're group settings. Since you can have multiple groups, I don't think it makes sense to move those to the admin section.

**Reply by jkosick** [reply]
> > It seems strange to have to use the profile page to access an admin settings
> 
> These aren't admin settings, they're group settings. Since you can have multiple groups, I don't think it makes sense to move those to the admin section.

Understood, but I still think if it's a group setting then you should be able to reach it by the group menu. I had been clicking around for quite some time and could not find that area. I never thought to click on the profile page to access it. I guess this setup would be to accommodate multiple people independently using the site? 

> I definitely think that makes sense, just wanted to point out that links to those pages do exist.
Thank you.
